KPCTabsControl/TabButton.swift

```diff
@@ -7,49 +7,69 @@
 //
 
 import AppKit
+fileprivate func < <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
```

Comment:
This comparator function is written 3 times, and the next one 2. That's not really something I want to keep like that. But I've tried to refactor those to `Helpers.swift`, and it fails to compile, because it seems to not be visible anymore in other files. Even if we drop `fileprivate`. Could you have a look?